it/firefox/products/lockwise.lang

```diff
@@ -0,0 +1,85 @@
+## NOTE: https://bedrock-demo-lockwise.oregon-b.moz.works/firefox/lockwise/
+## URL: https://bedrock-demo-lockwise.oregon-b.moz.works/%LOCALE%/firefox/lockwise/
+
+
+# HTML page title
+;Firefox Lockwise — password manager — take your passwords everywhere
+Firefox Lockwise — password manager — take your passwords everywhere
+
+
+# Page discription
+;Firefox Lockwise lets you securely access the passwords you’ve saved in Firefox from anywhere — even outside of the browser. Features 256-bit encryption and Face/Touch ID.
+Firefox Lockwise lets you securely access the passwords you’ve saved in Firefox from anywhere — even outside of the browser. Features 256-bit encryption and Face/Touch ID.
+
+
+# Main header
+;Take your passwords everywhere
+Take your passwords everywhere
+
+
+# Discription under header
+;Securely access the passwords you’ve saved in Firefox from anywhere — even outside of the browser.
+Securely access the passwords you’ve saved in Firefox from anywhere — even outside of the browser.
+
+
+# iTunes download button
+;Download on the App Store
+Download on the App Store
+
+
+# Google play download button
+;GET IT ON Google Play
+GET IT ON Google Play
+
+
+;Try the Lockwise add-on
+Try the Lockwise add-on
+
+
+# CTA button
+;Install for Firefox
+Install for Firefox
+
+
+# Lockwise refers to Firefox Lockwise. Do not translate.
+;Try Lockwise now
+Try Lockwise now
+
+
+;Only in the Firefox Browser
+Only in the Firefox Browser
+
+
+# CTA button
+;Get Firefox
+Get Firefox
+
+
+;256-bit encryption protects you while syncing
+256-bit encryption protects you while syncing
+
+
+;Get to your passwords securely with Face or Touch ID
+Get to your passwords securely with Face or Touch ID
+
+
+;Your privacy comes first.
+Your privacy comes first.
+
+
+;We keep your data safe, never sold.
+We keep your data safe, never sold.
+
+
+;Resources
+Resources
+
+
+# This is a link that points to https://support.mozilla.org/products/firefox-lockwise
+;Support
+Support
+
+
+# This is a link that points to https://lockwise.firefox.com/privacy.html
+;Your Privacy
+Your Privacy
```

Comment:
This file is definitely not generated via langchecker…

It would have an empty line at the end, and it wouldn't have the `## URL` comment (that's only in the source).

Reply:
I created it because of some confusion on the intent behind different options set for langchecker and how these files work in the system in general. I have since gotten some clarification on how this all works and will updating everything to be correct.